internal/build/package.json

```diff
@@ -33,7 +33,7 @@
     "test:coverage": "yarn test --coverage"
   },
   "dependencies": {
-    "@vertexvis/build-tools": "0.8.0",
+    "@vertexvis/build-tools": "0.8.1",
```

Comment:
TODO: This will need to be updated with the release of https://github.com/Vertexvis/vertex-web-tools/pull/28